### PR TITLE
Add SSG content for McAfee VSEL

### PR DESCRIPTION
**Pull request by cyarbrough76**
#### Description:

- _Add new content dir with checks and remediations. edit makelists, buil.product and constance.py_

#### Rationale:

- _Add content to identify and remediate non compliance with VSEL STIG requirements in_

### CODE REVIEW

**Comment by mildas** [comment]
**Changes identified:**
Rule checks:
 OVAL check is newly added.
 The rule doesn't occur in any profile nor product.
Rule virus_notification:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule web_client_disabled:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_action_app_primary:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_action_app_secondary:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_action_default_primary:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_action_default_secondary:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_action_error:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_action_timeout:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_allFiles:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_decompArchive:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_enabled:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_heuristicAnalysis:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_macroAnalysis:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_program:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_scanMaxTmo:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_scanNWFiles:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_scanOnRead:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_scanOnWrite:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_action_app_primary:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_action_app_secondary:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_action_default_primary:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_action_default_secondary:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_allFiles:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_decompArchive:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_extensions:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_heuristicAnalysis:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_macroAnalysis:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_mime:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_program:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_scanNWFiles_local:
 OVAL check is newly added.
 Bash remediation is newly added.
Profile stig on vsel:
 Newly added profile.
Others:
 Python abstract syntax tree change found in ssg/constants.py.

**Recommended tests to execute:**
 build_product vsel
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml virus_notification
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml web_client_disabled
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_action_app_primary
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_action_app_secondary
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_action_default_primary
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_action_default_secondary
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_action_error
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_action_timeout
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_allFiles
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_decompArchive
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_enabled
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_heuristicAnalysis
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_macroAnalysis
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_program
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_scanMaxTmo
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_scanNWFiles
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_scanOnRead
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_scanOnWrite
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_action_app_primary
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_action_app_secondary
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_action_default_primary
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_action_default_secondary
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_allFiles
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_decompArchive
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_extensions
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_heuristicAnalysis
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_macroAnalysis
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_mime
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_program
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_scanNWFiles_local
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-vsel-ds.xml stig
 (cd build && cmake ../ && ctest -j4)

**Comment by redhatrises** [comment]
@openscap-ci test this please

**Comment by mildas** [comment]
**Changes identified:**
Rule checks:
 OVAL check is newly added.
 The rule doesn't occur in any profile nor product.
Rule virus_notification:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule web_client_disabled:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_action_app_primary:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_action_app_secondary:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_action_default_primary:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_action_default_secondary:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_action_error:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_action_timeout:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_allFiles:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_decompArchive:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_enabled:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_heuristicAnalysis:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_macroAnalysis:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_program:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_scanMaxTmo:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_scanNWFiles:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_scanOnRead:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_scanOnWrite:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_action_app_primary:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_action_app_secondary:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_action_default_primary:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_action_default_secondary:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_allFiles:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_decompArchive:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_extensions:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_heuristicAnalysis:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_macroAnalysis:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_mime:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_program:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_scanNWFiles_local:
 OVAL check is newly added.
 Bash remediation is newly added.
Profile stig on vsel:
 Newly added profile.
Others:
 Python abstract syntax tree change found in ssg/constants.py.

**Recommended tests to execute:**
 (cd build && cmake ../ && ctest -j4)
 build_product vsel
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml virus_notification
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml web_client_disabled
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_action_app_primary
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_action_app_secondary
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_action_default_primary
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_action_default_secondary
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_action_error
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_action_timeout
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_allFiles
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_decompArchive
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_enabled
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_heuristicAnalysis
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_macroAnalysis
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_program
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_scanMaxTmo
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_scanNWFiles
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_scanOnRead
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_scanOnWrite
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_action_app_primary
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_action_app_secondary
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_action_default_primary
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_action_default_secondary
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_allFiles
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_decompArchive
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_extensions
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_heuristicAnalysis
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_macroAnalysis
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_mime
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_program
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_scanNWFiles_local
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-vsel-ds.xml stig

File: CMakeLists.txt [hunk 1]
```diff
@@ -89,6 +89,7 @@ option(SSG_PRODUCT_SLE15 "If enabled, the SLE15 SCAP content will be built" ${SS
 option(SSG_PRODUCT_UBUNTU1404 "If enabled, the Ubuntu 14.04 SCAP content will be built" ${SSG_PRODUCT_DEFAULT})
 option(SSG_PRODUCT_UBUNTU1604 "If enabled, the Ubuntu 16.04 SCAP content will be built" ${SSG_PRODUCT_DEFAULT})
 option(SSG_PRODUCT_UBUNTU1804 "If enabled, the Ubuntu 18.04 SCAP content will be built" ${SSG_PRODUCT_DEFAULT})
+option(SSG_PRODUCT_VSEL "If enabled, the McAfee VirusScan Enterprise for Linux SCAP content will be built" TRUE)
```
Comment (redhatrises) [inline CMakeLists.txt:92]: Let's change VirusScan Enterprise for Linux to VSEL

File: CMakeLists.txt [hunk 2]
```diff
@@ -262,6 +263,7 @@ message(STATUS "SUSE 15: ${SSG_PRODUCT_SLE15}")
 message(STATUS "Ubuntu 14.04: ${SSG_PRODUCT_UBUNTU1404}")
 message(STATUS "Ubuntu 16.04: ${SSG_PRODUCT_UBUNTU1604}")
 message(STATUS "Ubuntu 18.04: ${SSG_PRODUCT_UBUNTU1804}")
+message(STATUS "McAfee VirusScan Enterprise for Linux: ${SSG_PRODUCT_VSEL}")
```
Comment (redhatrises) [inline CMakeLists.txt:266]: Let's change VirusScan Enterprise for Linux to VSEL

**Comment by redhatrises** [comment]
@openscap-ci test this please

**Comment by mildas** [comment]
**Changes identified:**
Rule checks:
 OVAL check is newly added.
 The rule doesn't occur in any profile nor product.
Rule virus_notification:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule web_client_disabled:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_action_app_primary:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_action_app_secondary:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_action_default_primary:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_action_default_secondary:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_action_error:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_action_timeout:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_allFiles:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_decompArchive:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_enabled:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_heuristicAnalysis:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_macroAnalysis:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_program:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_scanMaxTmo:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_scanNWFiles:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_scanOnRead:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_scanOnWrite:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_action_app_primary:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_action_app_secondary:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_action_default_primary:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_action_default_secondary:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_allFiles:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_decompArchive:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_extensions:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_heuristicAnalysis:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_macroAnalysis:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_mime:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_program:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_scanNWFiles_local:
 OVAL check is newly added.
 Bash remediation is newly added.
Profile stig on vsel:
 Newly added profile.
Others:
 Python abstract syntax tree change found in ssg/constants.py.

**Recommended tests to execute:**
 (cd build && cmake ../ && ctest -j4)
 build_product vsel
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml virus_notification
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml web_client_disabled
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_action_app_primary
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_action_app_secondary
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_action_default_primary
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_action_default_secondary
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_action_error
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_action_timeout
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_allFiles
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_decompArchive
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_enabled
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_heuristicAnalysis
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_macroAnalysis
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_program
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_scanMaxTmo
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_scanNWFiles
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_scanOnRead
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_scanOnWrite
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_action_app_primary
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_action_app_secondary
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_action_default_primary
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_action_default_secondary
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_allFiles
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_decompArchive
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_extensions
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_heuristicAnalysis
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_macroAnalysis
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_mime
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_program
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_scanNWFiles_local
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-vsel-ds.xml stig

**Comment by redhatrises** [comment]
@cyarbrough76 are you also going to maintain this profile?

**Comment by mildas** [comment]
**Changes identified:**
Rule checks:
 The rule doesn't occur in any profile nor product.
 OVAL check is newly added.
Rule virus_notification:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule web_client_disabled:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_action_app_primary:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_action_app_secondary:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_action_default_primary:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_action_default_secondary:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_action_error:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_action_timeout:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_allFiles:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_decompArchive:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_enabled:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_heuristicAnalysis:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_macroAnalysis:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_program:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_scanMaxTmo:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_scanNWFiles:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_scanOnRead:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule oas_scanOnWrite:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_action_app_primary:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_action_app_secondary:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_action_default_primary:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_action_default_secondary:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_allFiles:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_decompArchive:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_extensions:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_heuristicAnalysis:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_macroAnalysis:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_mime:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_program:
 OVAL check is newly added.
 Bash remediation is newly added.
Rule ods_scanNWFiles_local:
 OVAL check is newly added.
 Bash remediation is newly added.
Profile stig on vsel:
 Newly added profile.
Others:
 Python abstract syntax tree change found in ssg/constants.py.

**Recommended tests to execute:**
 build_product vsel
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml virus_notification
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml web_client_disabled
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_action_app_primary
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_action_app_secondary
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_action_default_primary
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_action_default_secondary
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_action_error
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_action_timeout
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_allFiles
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_decompArchive
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_enabled
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_heuristicAnalysis
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_macroAnalysis
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_program
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_scanMaxTmo
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_scanNWFiles
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_scanOnRead
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml oas_scanOnWrite
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_action_app_primary
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_action_app_secondary
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_action_default_primary
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_action_default_secondary
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_allFiles
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_decompArchive
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_extensions
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_heuristicAnalysis
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_macroAnalysis
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_mime
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_program
 tests/test_suite.py rule --libvirt qemu:///system test-suite-vm --remediate-using bash --datastream build/ssg-vsel-ds.xml ods_scanNWFiles_local
 tests/test_suite.py profile --libvirt qemu:///system test-suite-vm --datastream build/ssg-vsel-ds.xml stig
 (cd build && cmake ../ && ctest -j4)

File: CMakeLists.txt (hunk 1 above)
Comment (cyarbrough76) [reply CMakeLists.txt]: fixed

File: CMakeLists.txt (hunk 2 above)
Comment (cyarbrough76) [reply CMakeLists.txt]: fixed

**Comment by cyarbrough76** [comment]
We intend to apply and test this content quarterly for the foreseeable future and will certainly submit any issues we find.

**Comment by cyarbrough76** [comment]
> 
> 
> @cyarbrough76 are you also going to maintain this profile?

We intend to apply and test this content quarterly for the foreseeable future and will certainly submit any issues we find.

**Comment by redhatrises** [comment]
> > @cyarbrough76 are you also going to maintain this profile?
> 
> We intend to apply and test this content quarterly for the foreseeable future and will certainly submit any issues we find.

Sweet! I'll merge, but always think about enhancing the content further for readability and multiple use applications. For example, the titles can be broadly readable that aren't STIG specific for submitting to NIST for review in the NCP program, etc.

**Pull request merged**